client/src/reducers/CartReducer.js

```diff
@@ -0,0 +1,113 @@
+import {
+  ADD_TO_CART,
+  DECREASE_COUNT,
+  DELETE_FROM_CART,
+  INCREASE_COUNT,
+  RESET_CART,
+  GET_CHOSEN_CHEF_PROFILE,
+} from "../constants/userConstants";
+
+export const cartInitialState = {
+  cart: localStorage.getItem("cart")
```

Comment:
This is nice, I will be adding ShippingAddress from localstorage inside the cartInitialState. I will keep the format same inside my cart reducer